initialize/services.go

```diff
@@ -133,7 +134,16 @@ func localIndexer(ctx context.Context, cfg *config.Config) (indexer.Service, err
 			opts.ScannerConfig.Repo[name] = node.Decode
 		}
 	}
-	s, err := libindex.New(ctx, &opts)
+	tr := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
whats the context around this change?

Reply:
New Libvuln API now requires an http.Client at construction. So this constructs a client via the config's method and then passes it in.